account_invoice_export/models/transmit_method.py

```diff
@@ -21,6 +21,14 @@ def get_transmission_http_header(self):
 
         """
         self.ensure_one()
-        auth = "{}:{}".format(self.destination_user, self.destination_pwd,)
+        auth = "{}:{}".format(self.destination_user or "", self.destination_pwd or "")
```

Comment:
wouldn't be better to raise a specific error?

Reply:
Well my thought was letting the call be made and let `request` or the distant server raise an error.
From what I know the password could be empty ?

Reply:
yeah but the error might be not so clear for the user. I guess destination user is mandatory? If yes, raising a UserError here or before should be better. Unless I miss something else.